src/app/shared/pipes/path/path.pipe.ts

```diff
@@ -0,0 +1,22 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { UrlHelperService } from '@app/core/url';
+
+export type PathType = 'basePath' | 'imagesPath';
+
+@Pipe({
+  name: 'path',
+})
+export class PathPipe implements PipeTransform {
+  constructor(protected urlHelperService: UrlHelperService) {}
+
+  public transform(url: string, type: PathType): string {
+    switch (type) {
+      case 'basePath':
+        return this.urlHelperService.basePath() + url;
```

Comment:
template literal?

Reply:
template literal ma sens, jak wrzucamy zmienną w string, aby nie robić takich rzeczy:
'Moje imię to ' + imieVar + ', a na drugie mam ' + drugieImie + '.' 

tutaj  łączymy wartość z metody + parametr i jest czytelniej w takiej formie